examples/cxx/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# Copyright: 2020 Fondazione Istituto Italiano di Tecnologia
+# CopyPolicy: Released under the terms of the MIT license
+
+cmake_minimum_required(VERSION 3.16)
```

Comment:
This is required also in the parent folder.

Reply:
It is already there, unless I am missing something.

Reply:
Sorry, I was indeed ambiguous. I meant that the same line was already present in the parent folder, so I was not sure whether this was needed.

Reply:
I think that for examples it is convenient that each directory can be built on its own, starting from the leaf directories up to the most parent (used by tests).

Reply:
Ah, I see. Ok then